meta-mender-demo/conf/layer.conf

```diff
@@ -11,3 +11,5 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 BBFILE_COLLECTIONS += "mender-demo"
 BBFILE_PATTERN_mender-demo = "^${LAYERDIR}/"
 BBFILE_PRIORITY_mender-demo = "6"
+
+MENDER_STORAGE_TOTAL_SIZE_MB ?= "300"
```

Comment:
We need `core-image-full-cmdline` for the tests though. Do you think it's alright to leave it at 300 for demo/test purposes?